packages/monitors/src/OptimisticOracleContractMonitor.js

```diff
@@ -304,6 +344,13 @@ class OptimisticOracleContractMonitor {
       }
     }
   }
+
+  _generateUILink(linkText, requester, identifier, timestamp, ancillaryData) {
+    const hexIdentifier = this.padRight(this.utf8ToHex(identifier), 64);
+    return `<${this.optimisticOracleUIBaseUrl}?chainId=${this.contractProps.chainId}&requester=${this.toChecksumAddress(
+      requester
+    )}&identifier=${hexIdentifier}&timestamp=${timestamp}&ancillaryData=${ancillaryData}|${linkText}>`;
```

Comment:
`${ancillaryData}|${linkText}`  this looks like its appending the link text after the link, is this so you can split it out of the logs later or what?

Reply:
This is how you tell slack to [do this](https://example.com). See here for where we do this for etherscan links: https://github.com/UMAprotocol/protocol/blob/6b53838d5d74d6f4b4f464ceefe1facd5ba16080/packages/common/src/FormattingUtils.ts#L112.

Reply:
ye, this is correct. for slack you need to create your links as:
```
<https://your-url.com|hyperlink>'
```

so what Matt's code is doing is building the link, then `|` then the "linkText` which will be `View this request in the UI.`.

my one main bit of feedback on this @mrice32 is why make the `linkText` a prop? you could hardcode it here as `View this request in the UI.`? Alternatively, make it the last prop in this function, default it to `View this request in the UI.` and then remove it from all the places you call the function.

Reply:
Yeah, we can hardcode it. I will make that change.

Reply:
Done.

Reply:
ye that makes the code cleaner. thanks!